geomesa-core/src/main/scala/org/locationtech/geomesa/core/index/QueryPlanners.scala

```diff
@@ -312,26 +313,28 @@ trait GeoHashPlanner extends Logging {
 }
 
 case class GeoHashKeyPlanner(offset: Int, bits: Int) extends KeyPlanner with GeoHashPlanner {
-  def getKeyPlan(filter: KeyPlanningFilter, output: ExplainerOutputType) = getKeyPlan(filter, offset, bits) match {
-    case KeyList(keys) =>
-      output(s"GeoHashKeyPlanner: ${keys.size} : ${keys.take(20)}")
-      KeyListTiered(keys)
+  def getKeyPlan(filter: KeyPlanningFilter, indexOnly: Boolean, output: ExplainerOutputType) =
+    getKeyPlan(filter, offset, bits) match {
+      case KeyList(keys) =>
+        output(s"GeoHashKeyPlanner: ${keys.size} : ${keys.take(20)}")
+        KeyListTiered(keys)
 
-    case KeyAccept =>
-      output(s"GeoHashKeyPlanner: KeyAccept")
-      KeyAccept
+      case KeyAccept =>
+        output(s"GeoHashKeyPlanner: KeyAccept")
+        KeyAccept
 
-    case _ => KeyInvalid
-  }
+      case _ => KeyInvalid
```

Comment:
They are aligned, it's just a bit hard to see in this view. If you look at the full file (non-diff) it's easier.